docs/api/odbc.md

```diff
@@ -0,0 +1,116 @@
+These assets have specific driver artifacts to the corresponding operating system.
+
+## Linux Setup
+The ODBC setup on Linux is based on files, the well-known **.odbc.ini** and **.odbcinst.ini**.
+These files can be placed at the system `/etc` directory or at the user home directory `/home/<user>` (shortcut as `~/`).
+The DM prioritizes the user configuration files and then the system files.
+
+### The ".odbc.ini" File
+
+The **.odbc.ini** contains the DSNs for the drivers, which can have specific knobs.
+
+An example of **.odbc.ini** with DuckDB would be:
+
+```
+[DuckDB]
+Driver = DuckDB Driver
+Database=:memory:
+```
+
+**[DuckDB]**: between the brackets is a DSN for the DuckDB.
+
+**Driver**: it describes the driver's name, and other configurations will be placed in the **.odbcinst.ini**.
+
+**Database**: it describes the database name used by DuckDB, and it can also be a file path to a `.db` in the system.
+
+### The ".odbcinst.ini" File
+
+The **.odbcinst.ini** contains general configurations for the ODBC installed drivers in the system.
+A driver section starts with the driver name between brackets, and then it follows specific configuration knobs belonging to that driver.
+
+An example of **.odbcinst.ini** with the DuckDB driver would be:
+
+```
+[ODBC]
+Trace = yes
+TraceFile = /tmp/odbctrace
+
+[DuckDB Driver]
+Driver = /home/<user>/duckdb/build/release/tools/odbc/libduckdb_odbc.so
+```
+
+
+**[ODBC]**: it is the DM configuration section.
+
+**Trace**: it enables the ODBC trace file using the option `yes`.
+
+**TraceFile**: the absolute system file path for the ODBC trace file.
+
+
+**[DuckDB Driver]**: the section of the DuckDB installed driver.
+
+**Driver**: the absolute system file path of the DuckDB driver. 
+
+
+
+## Windows Setup
+The ODBC setup on Windows is based on registry keys ([Registry Entries for ODBC Components
+](https://docs.microsoft.com/en-us/sql/odbc/reference/install/registry-entries-for-odbc-components?view=sql-server-ver15)).
+The ODBC entries can be placed at the current user registry key (`HKCU`) or the system registry key (`HKLM`).
+
+We have tested and used the system entries based on `HKLM->SOFTWARE->ODBC`.
+In that registry, there are two subkeys: `ODBC.INI` and `ODBCINST.INI`.
+
+The `ODBC.INI` is where users usually insert DSN registry entries for the drivers.
+
+For example, the DSN registry for DuckDB would look like this:
+
+![`HKLM->SOFTWARE->ODBC->ODBC.INI->DuckDB`](/images/blog/odbc/odbc_ini-registry-entry.png)
+
+
+The `ODBCINST.INI` contains one entry for each ODBC driver and other keys predefined for [Windows ODBC configuration](https://docs.microsoft.com/en-us/sql/odbc/reference/install/registry-entries-for-odbc-components?view=sql-server-ver15).
+
+### ODBC Windows Installer
+
+To simplify and facilitate the installation process, we provide the ODBC installer (`odbc_install.exe`) with the Windows asset.
```

Comment:
I would start off with the installer in the Windows setup section, as that seems to me like the easiest (and recommended?) way of installing ODBC for Windows users. The rest can be provided as more details.

Is it also possible to double click on the executable, or is running the command through the command prompt required? Windows users in general are less used to shells so double clicking would be easier.

Reply:
That sounds fine when launched via double click, but I would avoid launching message boxes when installing through a command line interface.

Reply:
I see, we have a /CI option already that disables message boxes.